synapse/config/server.py

```diff
@@ -319,6 +328,15 @@ def default_config(self, server_name, data_dir_path, **kwargs):
         #
         #use_presence: false
 
+        # whether to require users to authenticate in order to query /profile
```

Comment:
we should probably explain what `/profile` requests do, and why you might want to require auth?

Reply:
probably.